MekHQ/src/mekhq/gui/dialog/CustomizeScenarioDialog.java

```diff
@@ -331,11 +336,8 @@ public Component getListCellRendererComponent(final JList<?> list, final Object
                 BorderFactory.createEmptyBorder(5,5,5,5)));
 
         initOtherForcesPanel(resourceMap);
-        panOtherForces.setBorder(BorderFactory.createCompoundBorder(
-                BorderFactory.createEmptyBorder(0, 0, 10, 0),
-                BorderFactory.createTitledBorder(resourceMap.getString("panOtherForces.title"))));
-        panOtherForces.setPreferredSize(new Dimension(600,250));
-        panOtherForces.setMinimumSize(new Dimension(600,250));
+        panOtherForces.setPreferredSize(new Dimension(600,300));
```

Comment:
I think we'd talked previously about making the MekHQ UI more responsive; would it be possible to use a persistent size, or a % of screen size, or something other than magic numbers here?

Reply:
Looks like it already got merged, but I can think about the issue in future dialog stuff.